### PR TITLE
docs: add faq for pod autoscaling

### PR DESCRIPTION
**Pull request by jmdeal**
Fixes #N/A 

**Description**
Adds an FAQ about pod autoscaling being outside of Karpenter's scope and gives recommendations.

**How was this change tested?**
N/A

**Does this change impact docs?**
- [x] Yes, PR includes docs updates 
- [ ] Yes, issue opened: # 
- [ ] No

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: website/content/en/docs/faq.md [hunk 1]
```diff
@@ -185,6 +185,10 @@ amiSelectorTerms:
     - name: Windows_Server-2022-English-Full-EKS_Optimized-{{< param "latest_k8s_version" >}}*
 ```
 
+### Can I use Karpenter to scale my workloads?
```
Comment (jonathan-innis) [inline website/content/en/docs/faq.md:188]: I think this question is a little ambigouous. A natural question is here is "scale what?". Maybe something a little more deliberate might help here. Not exactly sure on the wording (we can iron it out) but maybe something like, "Can I use Karpenter to scale my container resources or replica count?"

Comment (jmdeal) [reply website/content/en/docs/faq.md:188]: Agree it's a bit ambiguous. Maybe "Can I use Karpenter to horizontally scale my workloads?" To me that means horizontal pod autoscaling but I'm not sure if that term is overloaded in any other way that still makes it ambiguous.

Comment (jmdeal) [reply website/content/en/docs/faq.md]: Discussed offline, settled on "Can I use Karpenter to scale my workload's pods" to encapsulate both vertical and horizontal pod autoscaling.

**Review by jonathan-innis — APPROVED** [review]
LGTM 🚀

**Pull request merged**